cmd/kubelet/app/server.go

```diff
@@ -361,15 +353,37 @@ func (s *KubeletServer) createAPIServerClient() (*client.Client, error) {
 	if len(s.APIServerList) > 1 {
 		glog.Infof("Multiple api servers specified.  Picking first one")
 	}
-	clientConfig.Host = s.APIServerList[0]
 
-	s.addChaosToClientConfig(&clientConfig)
+	// This creates a client, first loading any specified kubeconfig
+	// file, and then overriding the Master flag, if non-empty.
+	clientConfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
```

Comment:
The fork we use does have `Changed`: https://godoc.org/github.com/spf13/pflag#Flag.

A need for `Changed` doesn't come up often, so only those places that need it and want compatibility with other flag libraries use it.